### ISSUE TITLE
Autocomplete suggestions: insufficient color contrast ratio

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#3631]
In all the autocomplete suggestions, the suggestions list can be navigated with arrow keys, and when doing so the suggestions are highlighted with a blue background while the text becomes white:

![screen shot 2017-11-23 at 17 06 16](https://user-images.githubusercontent.com/1682452/33181834-f4094032-d071-11e7-9744-60d607c829a9.png)

This color contrast ratio is too low for accessibility (WCAG level AA requires at least 4.5:1) and it's actually just 3.02:1 see https://jdlsn.com/color/?type=hex&color=FFFFFF&color2=00a0d2

The colors should be adjusted to pass the required 4.5:1. In core there are a couple examples of autocomplete suggestions lists: the links and the tags:

![screen shot 2017-11-23 at 17 09 07](https://user-images.githubusercontent.com/1682452/33181935-3b2a0bfe-d072-11e7-9996-d19e71f9f5a7.png)

and they both have a sufficient color contrast.

Worth also noting when hovering the links suggestions results, there's no "highlighting" while the tag suggestions do highlight the results also on hover:

<img width="295" alt="screen shot 2017-11-23 at 17 18 51" src="https://user-images.githubusercontent.com/1682452/33182286-af51feaa-d073-11e7-85bb-d92732a02ce5.png">

Re: the tag suggestions, the matched string can't be indicated by color alone. I'd suggest to use an underline, that would also make finding a new colors combo easier.

The autocomplete in Gutenberg are actually different components, but the necessary change is just about CSS and should be easy to do in one go. Basically `#00a0d2` can't be used with white text, there's the need of a darker blue.
